src/screens/docs/components/victory-scatter/ecology.md

```diff
@@ -1,267 +1,466 @@
 # VictoryScatter
 
-VictoryScatter renders a dataset as a series of points. VictoryScatter can be composed with [VictoryChart] to create scatter plots.
+VictoryScatter renders a dataset as a series of points. VictoryScatter can be composed with [`VictoryChart`] to create scatter plots.
+
+```playground
+<VictoryChart
+  theme={VictoryTheme.material}
+  domain={{ x: [0, 5], y: [0, 7] }}
+  style={{
+    parent: { border: "1px solid #ccc"}
+  }}
+>
+  <VictoryScatter
+    style={{
+      data: { fill: "#c43a31" },
+      parent: { border: "1px solid #ccc"}
+    }}
+    size={7}
+    data={[
+      { x: 1, y: 2 },
+      { x: 2, y: 3 },
+      { x: 3, y: 5 },
+      { x: 4, y: 4 },
+      { x: 5, y: 7 }
+    ]}
+  />
+</VictoryChart>
+```
+
+## Props
+
+### `animate`
+
+`VictoryScatter` uses the standard `animate` prop. [Read about it here](https://formidable.com/open-source/victory/docs/common-props#animate)
+
+See the [Animations Guide] for more detail on animations and transitions
+
+```js
+  animate={{
+    duration: 2000,
+    onLoad: { duration: 1000 },
+    onEnter: { duration: 500, before: () => ({y: 0}) }
+  )}
+```
+### `bubbleProperty`
+
+The `bubbleProperty` prop indicates which property of the data object should be used to scale data points in a bubble chart. If a `bubbleProperty` is given, `size` and `symbol` props will be ignored. (Bubble charts should render circles).
```

Comment:
> (Bubble charts should render circles)

Does this mean bubble charts **will** render circles, or is the user responsible for making that happen?